tokio/src/runtime/thread_pool/tests/worker.rs

```diff
@@ -1,77 +0,0 @@
-use crate::runtime::blocking;
```

Comment:
These tests were originally written before the full pool could be tested. Other tests should cover the use cases.